lib/src/screens/restore/wallet_restore_from_keys_form.dart

```diff
@@ -37,6 +37,13 @@ class WalletRestoreFromKeysFromState extends State<WalletRestoreFromKeysFrom> {
   final TextEditingController viewKeyController;
   final TextEditingController spendKeyController;
   final TextEditingController nameTextEditingController;
+  bool buttonPressed;
+
+  @override
+  void initState() {
+    super.initState();
+    buttonPressed = false;
+  }
 
```

Comment:
What is doing that ?

Reply:
@mkyq , this is my mistake. I was trying to use variable to control onTap. Forgot to delete this one. I am deleting these lines.

Reply:
@RafiaChy ok